Libplanet/Store/DefaultStore.cs

```diff
@@ -817,11 +816,12 @@ public void Dispose()
             RawBlock rawBlock;
             try
             {
-                var formatter = new BencodexFormatter<RawBlock>();
                 using (Stream stream = _blocks.OpenFile(
                     path, System.IO.FileMode.Open, FileAccess.Read, FileShare.Read))
+                using (MemoryStream ms = new MemoryStream())
                 {
-                    rawBlock = (RawBlock)formatter.Deserialize(stream);
+                    stream.CopyTo(ms);
+                    rawBlock = new RawBlock(ms.ToArray());
```

Comment:
We also can use [`File.ReadAllBytes()`](https://docs.microsoft.com/en-us/dotnet/api/system.io.file.readallbytes?view=netstandard-2.0) instead of streams.